.drone.yml

```diff
@@ -98,6 +130,7 @@ steps:
       command_timeout: 600s
       script:
         - /srv/www/owf-staging/deploy.sh
+    depends_on: [ clone ]
```

Comment:
I think maybe deployment should depend on the project at least compiling correctly?

Reply:
Yeah, I was first under the impression that this dependency "cloned" all dependencies from the last job, but as I found no documentation of it, I added all steps manually.